internal/cri/io/exec_io.go

```diff
@@ -55,6 +55,15 @@ func NewFifoExecIO(id, root string, tty, stdin bool) (*ExecIO, error) {
 }
 
 // NewStreamExecIO creates exec io with streaming.
+// address should be in the form of
+// <ttrpc|grpc>+<unix|vsock|hvsock>://<uds-path|vsock-cid:vsock-port|uds-path:hvsock-port>
```

Comment:
I think we don't need to list all the combinations in here. Suggest to use `protocol://address?streaming_id=xyz`.

Reply:
done